### ISSUE TITLE
mycli crashs when it encounters invalid sql statement

### ISSUE THREAD

**Issue by leunggamciu** [dbcli/mycli#93]
```
mysql cookbook@localhost:cookbook> select * from v1 cross join v2 JOIN v1.id = Exception in thread Thread-48:
Traceback (most recent call last):                                                            
  File "/usr/lib/python2.7/threading.py", line 810, in __bootstrap_inner                      
    self.run()                                                                                
  File "/usr/lib/python2.7/threading.py", line 763, in run                                    
    self.__target(*self.__args, **self.__kwargs)                                              
  File "/usr/local/lib/python2.7/dist-packages/prompt_toolkit/interface.py", line 482, in run 
    CompleteEvent(text_inserted=True)))                                                       
  File "/usr/local/lib/python2.7/dist-packages/mycli/sqlcompleter.py", line 238, in get_completions                                                                                                                 
    suggestions = suggest_type(document.text, document.text_before_cursor)
  File "/usr/local/lib/python2.7/dist-packages/mycli/packages/completion_engine.py", line 85, in suggest_type
    full_text, identifier)
  File "/usr/local/lib/python2.7/dist-packages/mycli/packages/completion_engine.py", line 267, in suggest_based_on_last_token
    prev_keyword, text_before_cursor, full_text, identifier)
  File "/usr/local/lib/python2.7/dist-packages/mycli/packages/completion_engine.py", line 209, in suggest_based_on_last_token
    elif (token_v.endswith('join') and token.is_keyword) or (token_v in
AttributeError: 'unicode' object has no attribute 'is_keyword'
```
Version: 1.0.1
OS:Ubuntu14.04 lts

**Comment by amjith** [comment]
I'm able to reproduce it. Thank you for reporting it. I'll try and get this fixed soon. 

Sorry about the bug.

**Comment by amjith** [comment]
This bug is now fixed in master. I'll try to release a new version sometime this weekend. But until then you can install directly from master using pip as follows: 

```
pip install https://github.com/dbcli/mycli/archive/master.zip
```

**Comment by amjith** [comment]
I've released a new version 1.1.1 to pypi. Please upgrade. 

    pip install -U mycli

**Comment by leunggamciu** [comment]
awesome!
